osf/utils/auth.py

```diff
@@ -30,27 +30,21 @@ def get_user(email=None, password=None, verification_key=None):
     query_list = []
     if email:
         email = email.strip().lower()
-        query_list.append(Q('emails__address', 'eq', email) | Q('username', 'eq', email))
+        query_list.append(Q(emails__address=email) | Q(username=email))
     if password:
         password = password.strip()
         try:
-            query = query_list[0]
-            for query_part in query_list[1:]:
-                query = query & query_part
-            user = User.find_one(query)
+            user = User.objects.get(query_list[0])
```

Comment:
Is this equivalent to the old code? Looks like the old code was `&`-ing all the query parts.